hips/tiles/tile_meta.py

```diff
@@ -37,15 +55,47 @@ def __eq__(self, other: 'HipsTileMeta') -> bool:
 
     @property
     def path(self) -> Path:
-        """Return the default path for tile storage (`~pathlib.Path`)."""
+        """Default path for tile storage (`~pathlib.Path`)."""
         return Path('hips', 'tiles', 'tests', 'data')
 
     @property
     def filename(self) -> str:
-        """Return the filename of HiPS tile (`str`)."""
+        """Filename for HiPS tile (`str`)."""
         return ''.join(['Npix', str(self.ipix), '.', self.file_format])
 
     @property
     def full_path(self) -> Path:
         """Full path (folder and filename) (`~pathlib.Path`)"""
         return self.path / self.filename
+
+    @property
+    def nside(self) -> int:
+        """nside of the HEALPix map"""
+        return hp.order2nside(self.order)
+
+    @property
+    def dst(self) -> np.ndarray:
+        """Destination array for projective transform"""
+        return np.array(
+            [[self.tile_width - 1, 0],
+             [self.tile_width - 1, self.tile_width - 1],
+             [0, self.tile_width - 1],
+             [0, 0]])
+
+    @property
+    def skycoord_corners(self) -> SkyCoord:
+        """Corner values for a HiPS tile"""
+        theta, phi = boundaries(self.nside, self.ipix)
+        if self.frame == 'galactic':
```

Comment:
If you use the values that SkyCoord support for the `frame` attribute also on this class, then you don't need the if-else here.